pkg/ip/ip_test.go

```diff
@@ -751,27 +750,6 @@ func (s *IPTestSuite) TestKeepUniqueIPs(c *C) {
 	}
 }
 
-// Note: each "op" works on size things
-func (s *IPTestSuite) BenchmarkKeepUniqueIPs(c *C) {
-	size := 1000
-	ipsOrig := make([]net.IP, 0, size)
-	for i := 0; i < size; i++ {
-		ipsOrig = append(ipsOrig, net.IPv4(byte(i>>24), byte(i>>16), byte(i>>8), byte(i>>0)))
-	}
-	ips := make([]net.IP, 0, len(ipsOrig))
-
-	copy(ips, ipsOrig)
-	for i := 0; i < c.N; i++ {
-		c.StopTimer()
-		rand.Shuffle(len(ips), func(i, j int) {
-			ips[i], ips[j] = ips[j], ips[i]
-		})
-		c.StartTimer()
-
-		KeepUniqueIPs(ips)
-	}
-}
```

Comment:
I'm actually curious to see what the difference in benchmark results were between the old `KeepUniqueIPs` and the new `SortedUniqueFunc`.

Reply:
Here they are (I used the new benchmark to compare the old and the new function to see the differences while scaling the slice size):

```
name                  old time/op    new time/op    delta
KeepUniqueIPs/96-8      17.2µs ±34%    15.6µs ±29%    ~     (p=0.421 n=5+5)
KeepUniqueIPs/128-8     24.5µs ± 5%    23.8µs ± 3%    ~     (p=0.151 n=5+5)
KeepUniqueIPs/160-8     31.2µs ± 8%    29.8µs ± 2%  -4.47%  (p=0.032 n=5+5)
KeepUniqueIPs/192-8     37.2µs ± 4%    36.2µs ± 2%    ~     (p=0.056 n=5+5)
KeepUniqueIPs/256-8     50.6µs ± 1%    48.6µs ± 0%  -4.04%  (p=0.008 n=5+5)
KeepUniqueIPs/512-8      108µs ± 3%     105µs ± 1%  -3.27%  (p=0.008 n=5+5)
KeepUniqueIPs/1024-8     235µs ± 1%     225µs ± 0%  -4.26%  (p=0.008 n=5+5)

name                  old alloc/op   new alloc/op   delta
KeepUniqueIPs/96-8       96.0B ± 0%     96.0B ± 0%    ~     (all equal)
KeepUniqueIPs/128-8      96.0B ± 0%     96.0B ± 0%    ~     (all equal)
KeepUniqueIPs/160-8      96.0B ± 0%     96.0B ± 0%    ~     (all equal)
KeepUniqueIPs/192-8      96.0B ± 0%     96.0B ± 0%    ~     (all equal)
KeepUniqueIPs/256-8      96.0B ± 0%     96.0B ± 0%    ~     (all equal)
KeepUniqueIPs/512-8      96.0B ± 0%     96.0B ± 0%    ~     (all equal)
KeepUniqueIPs/1024-8     96.0B ± 0%     96.0B ± 0%    ~     (all equal)

name                  old allocs/op  new allocs/op  delta
KeepUniqueIPs/96-8        3.00 ± 0%      3.00 ± 0%    ~     (all equal)
KeepUniqueIPs/128-8       3.00 ± 0%      3.00 ± 0%    ~     (all equal)
KeepUniqueIPs/160-8       3.00 ± 0%      3.00 ± 0%    ~     (all equal)
KeepUniqueIPs/192-8       3.00 ± 0%      3.00 ± 0%    ~     (all equal)
KeepUniqueIPs/256-8       3.00 ± 0%      3.00 ± 0%    ~     (all equal)
KeepUniqueIPs/512-8       3.00 ± 0%      3.00 ± 0%    ~     (all equal)
KeepUniqueIPs/1024-8      3.00 ± 0%      3.00 ± 0%    ~     (all equal)
```

Reply:
Awesome, thanks. It would be great to have them in the commit msg IMO